app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,9 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link href='https://fonts.googleapis.com/css?family=Crimson Text' rel='stylesheet'>
```

Comment:
can we use an `@import` statement for these fonts and put them in the scss? that way the fonts get compiled and served in the stylesheet, which i think is a bit more performant.

```
@import url('https://fonts.googleapis.com/css?family=Crimson Text');
```

Reply:
Done!